tests/security/test_blobstorage.py

```diff
@@ -17,7 +17,7 @@ def test_blobstorage(config, sample_input, encrypt):
 def test_encoded_format(config, sample_input, encrypt):
     config["ENCRYPT_SECRETS"] = encrypt
     encoded = encode_blob(sample_input)
-    assert encoded.startswith(chr(encrypt) + "\x00\x00\x00\x00")
```

Comment:
This is a cryptic way of returning `"\x01"` and `"\x00"`. Since encrypt is `bool` and in Python `bool` is a subclass of `int`, `True` becomes `1` and `False` becomes `0`.